### PR TITLE
[1LP][RFR]Enabled test for 5.9 and added support for cloud providers

### PR DESCRIPTION
**Pull request by sshveta**
{{pytest: -v --long-running --use-provider openstack cfme/tests/ssui/test_ssui_myservice.py -k test_myservice_crud }}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit bef95844162ee01062cd120ebc7883e8fa76fcd2:

`cfme/services/myservice/ssui.py`:
- :bangbang: Line 143:37: **E241** *multiple spaces after ','*

`widgetastic_manageiq/__init__.py`:
- :bangbang: Line 1012:101: **E501** *line too long (109 > 100 characters)*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/services/myservice/ssui.py [hunk 1]
```diff
@@ -140,14 +140,10 @@ def is_displayed(self):
 
 @MiqImplementationContext.external_for(MyService.update, ViaSSUI)
 def update(self, updates):
-    view = navigate_to(self, 'Edit')
+    view = navigate_to(self, 'Edit', wait_for_view="True")
```
Comment (mkoura) [inline cfme/services/myservice/ssui.py]: **REQUIRED**: use ``wait_for_view=True`` (boolean). Applies to other places where this is used as well.

File: cfme/tests/ssui/test_ssui_myservice.py [hunk 2]
```diff
@@ -18,12 +19,14 @@
...
     pytest.mark.long_running,
-    pytest.mark.ignore_stream("upstream", "5.9"),
     pytest.mark.provider([InfraProvider],
                          required_fields=[['provisioning', 'template'],
                                           ['provisioning', 'host'],
                                           ['provisioning', 'datastore']],
                          scope="module"),
+    pytest.mark.provider([CloudProvider],
+                         required_fields=[['provisioning', 'image']],
+                         scope="module"),
```
Comment (mkoura) [inline cfme/tests/ssui/test_ssui_myservice.py]: **REQUIRED**: You can't use ``pytest.mark.provider`` twice, only the later one will be used. You need to use smtg like:
```
infra_prov = ProviderFilter(classes=[InfraProvider], required_fields=[...])
cloud_prov = ProviderFilter(classes=[CloudProvider], required_fields=[...])

pytestmark = [
    ...
    pytest.mark.provider(gen_func=providers, filters=[infra_prov, cloud_prov], scope='module'),
```

File: cfme/tests/ssui/test_ssui_myservice.py [hunk 3]
```diff
@@ -2,6 +2,7 @@
 """Test Service Details page functionality."""
 import pytest
 
+from cfme.cloud.provider import CloudProvider
 from cfme.infrastructure.provider import InfraProvider
 from cfme.services.myservice import MyService
 from cfme import test_requirements
```
Comment (mkoura) [inline cfme/tests/ssui/test_ssui_myservice.py]: OPTIONAL: nice opportunity to sort imports

**Comment by sshveta** [comment]
PR will fail on 5.9 as one change is required in 
https://github.com/RedHatQE/widgetastic.patternfly/blob/master/src/widgetastic_patternfly/__init__.py#L763

BY_PARTIAL_VISIBLE_TEXT = (
'.//div/ul/li/a[./span[contains(@class, "text") and contains(normalize-space(.), {})]]') needs to be changed to 

BY_PARTIAL_VISIBLE_TEXT = (
'//div/ul/li/a[./span[contains(@class, "text") and contains(normalize-space(.), {})]]')

@mfalesni : can you please change this ? In 5.9 it is not working with (dot)

**Pull request merged**